js/src/components/pmax-improve-assets-banner/banner.js

```diff
@@ -26,10 +52,20 @@ import './index.scss';
  * @param {Function} props.onBannerDismissed Callback function to call when the banner is dismissed.
  *
  * @return {JSX.Element|null} The banner component, or null if not applicable.
+ *
+ * @fires gla_pmax_assets_improvements_banner_shown when the banner is displayed.
+ * @fires gla_pmax_assets_improvements_improve_assets_clicked when the "Improve Assets" button is clicked.
+ * @fires gla_pmax_assets_improvements_dismiss_clicked when the banner is dismissed.
  */
 const Banner = ( { onBannerDismissed } ) => {
 	const { campaign, hasFinishedResolution } = useRecommendedPMaxCampaign();
 
+	useEffect( () => {
+		recordGlaEvent( 'gla_pmax_assets_improvements_banner_shown', {
```

Comment:
@ankitrox This event will be fired even though there's no campaign because on L69 we are not displaying the banner if there's no campaign.